app/views/courses/show.html.erb

```diff
@@ -80,7 +80,7 @@
         <%= render partial: 'courses/about_course' %>
       <% end %>
 
-      <% if course.how_school_placements_work.present? %>
+      <% if course.how_school_placements_work.present? || course.program_type == 'higher_education_programme' || course.program_type == 'scitt_programme' %>
```

Comment:
Given the partial already includes logic for `course.program_type`, not sure it’s needed again here.

Reply:
If a course is a HE or Scitt program then we want the section to render the gray box with the relevant content.

However, we then need to guard against  `how_school_placements_work` being nil or it breaks. 

The reason we have to somewhat duplicate the logic called is if we remove
`<% if course.how_school_placements_work.present? %>` altogether, then it just renders the title and nothing else for school direct courses with no value for `course.how_school_placements_work.present?`

Reply:
let me try and phrase that better

Reply:
That didn't make loads of sense, but essentially sometimes we want to render our content even if `course.how_school_placements_work.present? == nil` (in the case of SCITT or HE courses). 

But we don't want to render the section if none of these conditions are true. Otherwise, we just end up with the title and nothing else.

Reply:
Ah, cool. In which case, :shipit: